locale/po/grasswxpy_fr.po

```diff
@@ -22505,11 +22505,6 @@ msgstr "étiquettes :"
 #~ msgid "Deactive overwrite"
```

Comment:
```suggestion
#~ msgid "Deactivate overwrite"
```

Reply:
The msgid must come from the template file, where all the source strings appear, that are collected from somewhere in our codebase. So just changing here, (in a translated file), will probably just break up things...

Reply:
I see...

Reply:
Once someone can approve I will!